packages/flutter/lib/src/material/slider.dart

```diff
@@ -91,13 +91,13 @@ class Slider extends StatefulWidget {
     this.label,
     this.activeColor,
     this.inactiveColor,
-  }) : assert(value != null),
-       assert(min != null),
-       assert(max != null),
-       assert(min <= max),
-       assert(value >= min && value <= max),
-       assert(divisions == null || divisions > 0),
-       super(key: key);
+  })  : assert(value != null),
```

Comment:
An extra space was inserted here, and on the following lines. This seems to be a systematic problem (dartfmt?). The result doesn't match the rest of the code base.

Reply:
Yeah, dartfmt recently "fixed" one of the bugs I wanted them to, but apparently made the decision that indents shouldn't have odd numbers of spaces.  I've pointed it out to them (and fixed it here).

Reply:
FYI, here's Bob's response: https://github.com/dart-lang/dart_style/issues/658#issuecomment-370552389